akkeeper/src/main/scala/akkeeper/common/Controllers.scala

```diff
@@ -49,10 +53,16 @@ trait BaseController extends Directives with SprayJsonSupport with CommonApiJson
     }
   }
 
-  final protected def handleRequest(service: ActorRef, msg: Any)
+  final protected def handleRequest(msg: Any)
                                    (implicit timeout: Timeout): Route = {
     val result = service ? msg
     onSuccess(result)(applyHandlers)
   }
 }
 
+case class ControllerComposite(basePath: String,
```

Comment:
IMO `ControllerComposite` don't belong to the same file where the base interfaces are stored. Can we please keep it in its original file (together with corresponding tests).